thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/client/comparison/TimeOnTimeResponseParser.java

```diff
@@ -321,7 +322,10 @@ private void parseGroupByTimeDimensionResponse() {
           // Add only a non zero metric
           boolean addMetric = false;
           for (Row.Metric metric : otherBuilder.row.getMetrics()) {
-            if (metric.getCurrentValue() > 0.0 || metric.getBaselineValue() > 0.0) {
+            // If absolute percentage change is more than default threshold for OTHER
+            if (Utils
```

Comment:
I have been thinking about it. Tests require setting up some boilerplate code (Datagenerator etc). I would do it as separate commit.